.github/workflows/ci_test.yml

```diff
@@ -95,3 +95,32 @@ jobs:
       - run: pip install pygithub click
       - name: Check that the CODEOWNERS is valid
         run: python .github/workflows/notify_codeowners.py .github/CODEOWNERS
+  nbfmt:
+    name: Notebook format
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v1
+    - uses: actions/checkout@v2
+    - name: Install tensorflow-docs
+      run: python3 -m pip install -U git+https://github.com/tensorflow/docs
+    - name: Check notebook formatting
+      run: |
+        # Run on all notebooks to prevent upstream change.
+        echo "Check formatting with nbfmt:"
+        python3 -m tensorflow_docs.tools.nbfmt --test \
+            $(find docs/tutorials/ -type f -name *.ipynb)
+  nblint:
+    name: Notebook lint
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/setup-python@v1
+    - uses: actions/checkout@v2
+    - name: Install tensorflow-docs
+      run: python3 -m pip install -U git+https://github.com/tensorflow/docs
+    - name: Lint notebooks
+      run: |
+        # Run on all notebooks to prevent upstream change.
+        echo "Lint check with nblint:"
+        python3 -m tensorflow_docs.tools.nblint \
```

Comment:
Cool.
FYI, by default it will use the "tensorflow" and "google" style modules [over here](https://github.com/tensorflow/docs/tree/master/tools/tensorflow_docs/tools/nblint/style). Still building these out, but if you want anything custom you're welcome to create a `tensorflow_addons.py` module.

`nbfmt` is pretty well tested. `nblint` is still experimental.

Reply:
Sure. We'll take a look to see if we need some customization.